resources/js/Pages/Collections/View.tsx

```diff
@@ -169,7 +170,7 @@ const CollectionsView = ({
     const tabChangeHandler = (tab: TabName): void => {
         setSelectedTab(tab);
 
-        setFilterIsDirty(true);
+        // setFilterIsDirty(true);
```

Comment:
can you comment on this change for context, I forgot the issue with it

Reply:
When the tab changes this function set filter is dirty flag true which causes request to backend  and reloads the page with the new parameters which is irrelevant for the articles as I am loading data in the client side - making another request is useless.